USFMToolsSharp/HtmlRenderer.cs

```diff
@@ -177,6 +197,7 @@ private string RenderMarker(Marker input)
                     output.AppendLine("</div>");
                     break;
                 case MTMarker mTMarker:
+                    currentChapterLabel = null;

```

Comment:
I decided to do IDMarker, because there are two markers for the table of contents (\toc and \toca). This way I'll only have to write one line of code. :)

Reply:
That should be fine.